# Functional genomic analysis of genes important for *Candida albicans* fitness in diverse environmental conditions

## Overview

This repository contains data, Jupyter notebooks, and source code for conducting a functional genomic analysis of genes important for *Candida albicans* fitness in diverse environmental conditions. The analysis was performed using Python and R.

## Project Structure

The repository is organized as follows:

- `data/`: Contains the datasets and relevant data files used in the analysis.
- `notebooks/`: Jupyter notebooks for step-by-step data analysis, visualization, and interpretation.
- `src/`: Source code for Python and R scripts used in the analysis.

## Prerequisites & Installation

To run the analysis and reproduce the results, you'll need the following software and packages installed:

### Python

   * Python 3.9
   * numpy (1.20.3)
   * pandas (1.3.4)
   * matplotlib (3.4.3)
   * seaborn (0.11.2)
   * scipy (1.10.1)

### R

   * limma
   * pheatmap
   * RColorBrewer
   * plotrix
   * FLEX (https://github.com/csbio/FLEX_R)
   * ggplot2
   * ggthemes
   * gplots
   * ggrepel
   * reshape2
   * GGally
   * dplyr
   * tidyr
   * ggvenn

### Sequence alignment

   * See `src/barseq_to_reads/barseq_assignreads.txt` for details


## License

Distributed under the MIT License. See `LICENSE` for more information.



## Contact

Please email to Xiang Zhang (zhan6668@umn.edu) if you have any questions, comments, or suggestions regarding this program.

Project Link: https://github.com/csbio/C.albicans_in_vitro_pipeline
